nixos/modules/services/databases/influxdb2.nix

```diff
@@ -25,7 +25,6 @@ let
     optional
     subtractLists
     types
-    unique
```

Comment:
This has nothing to do with "fix service startup reliability". Separate commit or remove please?

Reply:
I didn't want to create two commits for two single line changes. I removed it now.